packages/web3-shared/base/src/utils/balance.ts

```diff
@@ -7,7 +7,7 @@ export function formatBalance(
     significant = decimals,
     isPrecise = false,
 ) {
-    let balance = new BigNumber(rawValue)
+    let balance = new BigNumber(rawValue).integerValue()
```

Comment:
It's risky to change a widely used method. Therefore, please update unit test cases related to your new changes.